packages/miniflare/test/plugins/kv/assets.spec.ts

```diff
@@ -0,0 +1,220 @@
+import assert from "assert";
+import fs from "fs/promises";
+import path from "path";
+import anyTest, { Macro, TestFn } from "ava";
+import esbuild from "esbuild";
+import { Miniflare } from "miniflare";
+import { useTmp } from "../../test-shared";
+
+const FIXTURES_PATH = path.resolve(
+	__dirname,
+	"..",
+	"..",
+	"..",
+	"..",
+	"test",
+	"fixtures",
+	"assets"
+);
```

Comment:
NIT: I prefer this to be:

```suggestion
const FIXTURES_PATH = path.resolve(__dirname, "../../../../test/fixtures/assets");
```